src/components/modal/ModalDialog.js

```diff
@@ -49,6 +64,24 @@ class ModalDialog extends React.Component<Props> {
 
     modalID: string = uniqueId('modal');
 
+    /**
+     * Renders a button if onBackButtonClick is passed in
+     * @return {ReactElement|null} - Returns the button, or null if the button shouldn't be rendered
+     */
+    renderBackButton() {
+        const { backButtonClick } = this.props;
```

Comment:
We typically use `on` and `handle` for prop name and internal handlers, respectively. So, in the parent component you'd want to use `onBackClick={handleBackClick}`. Can we rename this to `onBackClick`?